cfgov/unprocessed/js/modules/util/web-storage-proxy.js

```diff
@@ -81,6 +81,7 @@ function setStorage( storage ) {
   _storage = storage;
 }
 
+/* istanbul ignore next */
```

Comment:
Not sure how to test that this still works above the jsdoc. @KimberlyMunoz do you know how to check this?